api/src/legacy_soap_api/legacy_soap_api_schemas.py

```diff
@@ -1,3 +1,5 @@
+from typing import Iterator, List
```

Comment:
Can use `list`, no need for typing List anymore.

```suggestion
from typing import Iterator
```